schema/defs.json

```diff
@@ -165,6 +176,98 @@
         },
         "annotations": {
             "$ref": "#/definitions/mapStringString"
+        },
+        "LandlockFSAction": {
+            "type": "string",
+            "enum": [
+                "execute",
+                "write_file",
+                "read_file",
+                "read_dir",
+                "remove_dir",
+                "remove_file",
+                "make_char",
+                "make_dir",
+                "make_reg",
+                "make_sock",
+                "make_fifo",
+                "make_block",
+                "make_sym",
+                "refer",
+                "truncate"
+            ]
+        },
+        "LandlockNetworkAction": {
+            "type": "string",
+            "enum": [
+                "bind",
+                "connect"
+            ]
+        },
+        "ArrayOfLandlockFSActions": {
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/LandlockFSAction"
+            }
+        },
+        "ArrayOfLandlockNetworkActions": {
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/LandlockNetworkAction"
+            }
+        },
+        "LandlockRuleset": {
+            "type": "object",
+             "properties": {
+                 "handledAccessFS": {
+                     "$ref": "#/definitions/ArrayOfLandlockFSActions"
+                 },
+                 "handledAssessNetwork": {
+                      "$ref": "#/definitions/ArrayOfLandlockNetworkActions"
+                 }
+             }
+        },
+        "LandlockRulePathBeneath": {
+            "type": "object",
+            "properties": {
+                "allowedAccess": {
+                    "$ref": "#/definitions/ArrayOfLandlockFSActions"
+                },
+                "paths": {
+                    "$ref": "#/definitions/ArrayOfStrings"
+                }
+            }
+        },
+        "LandlockRulePortBeneath": {
+            "type": "object",
+            "properties": {
+                "allowedAccess": {
+                    "$ref": "#/definitions/ArrayOfLandlockNetworkActions"
+                },
+                "paths": {
+                    "$ref": "#/definitions/ArrayOfPorts"
+                }
+            }
+        },
+        "ArrayOfLandlockRulePathBeneaths": {
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/LandlockRulePathBeneath"
+            }
+        },
+        "ArrayOfLandlockRulePortBeneaths": {
```

Comment:
See comment on the other file about the word "beneath" when talking about network ports